.github/workflows/publish-release.yml

```diff
@@ -6,8 +6,8 @@ on:
 
 jobs:
   is-release:
-    # release merge commits come from github-actions[bot]
-    if: github.event.commits[0].author.name == 'github-actions[bot]'
+    # release merge commits come from github-actions
+    if: startsWith(github.event.commits[0].author.name, 'github-actions')
```

Comment:
What are the possible values for the author name that we care about? `github-actions` and `github-actions[bot]`, or something else?

Reply:
yep! it's exactly those two. you get `github-actions[bot]` in the squash+merge case and `github-actions` if you simply merge.
![image](https://user-images.githubusercontent.com/675259/184905053-62bacfa7-cc1f-4598-bc14-165414482ae5.png)
![image](https://user-images.githubusercontent.com/675259/184905007-d1c98e87-3654-4c17-8235-0920298488cf.png)